app/models/user.rb

```diff
@@ -1,6 +1,9 @@
 class User < ApplicationRecord
+  has_secure_password
+
   validates_presence_of :first_name, :last_name, :email
   validates :member, exclusion: { :in => [nil], :message => "can't be blank" }
   validates_uniqueness_of :email
+  validates :password, length: { minimum: 6 }
```

Comment:
Validating the length of the password is great, definitely an industry standard. We should talk about adding a requirement for a capital letter, special character, etc.